api/src/modules/h3-data/h3-data.controller.ts

```diff
@@ -34,7 +34,7 @@ export class H3DataController {
   async geth3ByIdAndResolution(
     @Query(ValidationPipe)
     queryParams: MaterialH3ByResolutionDto,
-  ): Promise<H3IndexValueData> {
+  ): Promise<Array<H3IndexValueData>> {
```

Comment:
I think you can also declare this as Promise<H3IndexValueData[]> 
This is FYI , no need to change it

Reply:
Totally correct, thanks for spotting it out. Will change it to keep code consistency and avoid my future self haunting me from the future.

Cheers!